basic/mobile_io/2c_feedback_orientation.py

```diff
@@ -0,0 +1,78 @@
+import hebi
+from time import sleep, time
+from matplotlib import pyplot as plt
+from mpl_toolkits import mplot3d
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+lookup = hebi.Lookup()
+
+# Wait 2 seconds for the module list to populate
+sleep(2.0)
+
+family_name = "HEBI" 
+module_name = "Mobile IO"
+
+group = lookup.get_group_from_names([family_name], [module_name])
+
+if group is None:
+  print('Group not found: Did you forget to set the module family and name above?')
+  exit(1)
+
+# Live Visualization
+# Starts logging in the background. Note that logging can be enabled at any time, and that it does not negatively
+# affect the performance of your running programs.
+group.start_log('dir', 'logs', mkdirs=True)
+
+print('Visualizing 3-DoF orientation estimate from the mobile device.')
+print('Move it around to make the feedback interesting...')
+
+plt.ion()
+f = plt.figure()
+
+
+
+duration = 10.0
+start_time = time()
+end_time = start_time + duration
+current_time = start_time
+
+while current_time < end_time:
+  current_time = time()
+  fbk = group.get_next_feedback()
+  orient = fbk[0].ar_orientation
+  
+  r = R.from_quat(orient)
+  rot_mat = r.as_matrix()
```

Comment:
We have a function (`util.math_utils.quat2rot`) to do this, and I'd prefer not to bring in a dependency like `scipy` (I forget if it's a different package than `numpy` or what...). However... it's not imported here (you need to add a directory to the `PATH` for Python to find it), so this warrants a discussion on the best way to resolve this dependency.

Reply:
@CalZim -- I agree we should use our existing library functions where possible.
@rLinks234 -- I assume you mean the import is for the scipy/numpy, or is that for ours?